src/TLuaInterpreter.cpp

```diff
@@ -12499,12 +12499,9 @@ void TLuaInterpreter::initIndenterGlobals()
                       .toUtf8().constData());
 
 #elif defined(Q_OS_UNIX)
-    // Need to tweak the lua path for the installed *nix case and to allow
-    // running from a shadow build directory, the latter means we HAVE to rename
-    // where the module code is stored or use a symbolic link from "lcf" to
-    // "lua_code_formatter" in the "3rdparty" source directory - a rename was
-    // choosen.
-    luaL_dostring(pIndenterState, QStringLiteral("package.path = '" LUA_DEFAULT_PATH "/?.lua;%1/../3rdparty/?.lua;' .. package.path")
+    // Need to tweak the lua path for the installed *nix case and AppImage builds as well as
+    // to allow running from a shadow build directory (both qmake and cmake).
+    luaL_dostring(pIndenterState, QStringLiteral("package.path = '" LUA_DEFAULT_PATH "/?.lua;%1/?.lua;%1/../3rdparty/?.lua;%1/../../3rdparty/?.lua;' .. package.path")
```

Comment:
This is the *nix one - but I've found we need something for *shadow builds* in the Windows build from source code case as well but you need to quadruple escaped the back-slashes to enable them to get past two C/C++ compiler passes one being the C++ one for TLuaInterpreter compilation; the other the Lua script one (the `QDir::toNativeSeparators(...)` is not enough for that as it only converts '/' to '\\') so to allow parallel Windows builds in a `build` or with the combination of qmake `CONFIG` values that occur on that platform, in `build/release` or `build/debug` directories, you also need a:

    #elif defined(Q_OS_WIN32)
         luaL_dostring(pIndenterState, QStringLiteral("package.path = %1\\\\?.lua;%1\\\\..\\\\3rdparty\\\\?.lua;%1\\\\..\\\\..\\\\3rdparty\\\\?.lua;' .. package.path")
    #endif

Reply:
I'll leave that for another PR.